core.py

```diff
@@ -149,10 +204,6 @@ def read_sensor(self, pin):
         return sensor_value
 
     def stop(self):
-        if self.arduino:
-            self.arduino.direct_micros(self.LEFT_MID, self.RIGHT_MID)
-        else:
-            self.PWMservo.set_servo(LEFT_SERVO_PIN, self.LEFT_MID)
-            self.PWMservo.set_servo(RIGHT_SERVO_PIN, self.RIGHT_MID)
-            for pin in range(0, 3):
-                i2c_lidar.turnoff(LIDAR_PINS[pin])
+        self.set_neutral()
+        for pin in range(0, 3):
```

Comment:
is `LIDAR_PINS` the list of pins? Then you can:
```
for pin in LIDAR_PINS:
    i2c_lidar.turnoff(LIDAR_PINS[pin])
```
or, for bonus single line points:
`[i2c_lidar.turnoff(LIDAR_PINS[pin]) for pin in LIDAR_PINS]`